modules/apps/portal-vulcan/portal-vulcan-api/src/main/java/com/liferay/portal/vulcan/batch/engine/BatchStrategy.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.vulcan.batch.engine;
+
+import com.liferay.batch.engine.strategy.ImportStrategy;
```

Comment:
I think this import would force the module clients to add the batch-api dependency with the vulcan-api dependency.

I would prefer to have an interface BatchStrategy here without any references to batch in the API module and then have an Impl BatchStrategyImpl class in the impl module adapting the Import strategy.

Checking now, I realized that we already have a dependency in the portal-vulcan-api build.gradle file to the batch-engine-api but it is unnecessary and we should get rid of it